example/functions/fromYaml/functions.yaml

```diff
@@ -3,4 +3,4 @@ kind: Function
 metadata:
   name: function-go-templating
 spec:
-  package: xpkg.upbound.io/crossplane-contrib/function-go-templating:latest
\ No newline at end of file
+  package: xpkg.upbound.io/crossplane-contrib/function-go-templating:v0.3.0
```

Comment:
Looks like there are some end of file problems?

Reply:
fixed 👍